vendor/bat-native-ads/include/bat/ads/ads.h

```diff
@@ -65,6 +65,11 @@ class ADS_EXPORT Ads {
   Ads() = default;
   virtual ~Ads() = default;
 
+  enum AdHistoryFilterType : int8_t {
+    NONE,
+    CONFIRMATION_TYPE_TRUMPING_POLICY

```

Comment:
Please use `kNone` and `kConfirmationTypeTrumpingPolicy`

Reply:
I thought the form was UPPER so followed that - I prefer the 'kSomething' myself.

Reply:
legacy code users uppercase, however google standards changed this so for new code pref k

Reply:
Done